src/renderer/store/handlers/contacts.ts

```diff
@@ -41,19 +41,37 @@ export interface ContactsStore {
 const initialState: ContactsStore = {}
 
 const setMessages = createAction<{
-  messages: {
+  messages:
+  | {
     [key: string]: DisplayableMessage
-  } | DisplayableMessage[]
+  }
+  | DisplayableMessage[]
   contactAddress: string
   username: string
   key: string
 }>(actionTypes.SET_DIRECT_MESSAGES)
+const setChannelMessages = createAction<{
+  messages:
+  | {
+    [key: string]: DisplayableMessage
+  }
+  | DisplayableMessage[]
+  contactAddress: string
+  username: string
+  key: string
+}>(actionTypes.SET_CHANNEL_MESSAGES)
 const addContact = createAction<{
   offerId?: string
   contactAddress: string
   username: string
   key: string
 }>(actionTypes.ADD_CONTACT)
+const addDirectContact = createAction<{
+  offerId?: string
```

Comment:
We don't need to worry about offerId anymore, do we?

Reply:
We don't, and we also do not need many others, but it's separate task to refactor those structures, it's better to avoid doing side stuff when working on specific task.